setup_osm.sql

```diff
@@ -0,0 +1,39 @@
+CREATE ROLE gs_role WITH LOGIN PASSWORD 'gs_pass';
+CREATE ROLE geoserver WITH LOGIN PASSWORD 'QO7mqieY784Y';
```

Comment:
no passwords on Github please. Esecially public repos

Reply:
> no passwords on Github please. Esecially public repos

fixed in https://github.com/geosolutions-it/docker-postgis/pull/3/commits/9bc3d3bc135d470e1f16c679c41c2b7c87920523